compiler/src/main/java/com/squareup/anvil/compiler/codegen/BindingModuleGenerator.kt

```diff
@@ -182,31 +181,20 @@ internal class BindingModuleGenerator(
       //
       // We precompute this list here and share the result in the methods below. Resolving classes
       // and types can be an expensive operation, so avoid doing it twice.
-      val bindingsReplacedInDaggerModules = contributedModuleAndInterfaceClasses
+      val bindingsReplacedInDaggerModules = contributedModuleClasses
         .asSequence()
-        .mapNotNull {
-          if (it.isAnvilModule()) {
-            // The generated Anvil Dagger module cannot replace bindings. Exclude it, because the
-            // descriptor can't be resolved since it's a generated class.
-            null
-          } else {
-            it.requireClassDescriptor(module)
```

Comment:
That was causing a bug. If another code generator generated a Dagger module, then we cannot resolve the descriptor for that. The reason is that we simulate multiple rounds of code generation. There is no API to pass to generated code to the compiler after each round. We can do this only once after all code has been generated.